src/mono/mono/utils/lifo-semaphore.c

```diff
@@ -23,7 +23,7 @@ mono_lifo_semaphore_delete (LifoSemaphore *semaphore)
 int32_t
 mono_lifo_semaphore_timed_wait (LifoSemaphore *semaphore, int32_t timeout_ms)
 {
-	LifoSemaphoreWaitEntry wait_entry = { 0 };
+	LifoSemaphoreWaitEntry wait_entry = {{0}};
```

Comment:
If we're changing the struct order, this change is unneeded (and will cause C++ builds to fail).

Just remove this from the diff and I'll approve/merge. Sorry for all the trouble.